libnetdata/log/log.c

```diff
@@ -624,7 +624,7 @@ void open_all_log_files() {
 // error log throttling
 
 time_t error_log_throttle_period = 1200;
-unsigned long error_log_errors_per_period = 200;
+unsigned long error_log_errors_per_period = 99999;
```

Comment:
@vkalintiris Forgot to reset the error log per period limit :upside_down_face: .